cogs/helper.py

```diff
@@ -1,40 +1,103 @@
 import discord
 from discord.ext import commands
+from utils.config import subjects 
 
 
 class Helper(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
-        self.bot = bot
-
+        self.bot = bot        
+        self.subjects = dict(map(int, pair.split(',')) for pair in subjects.split(';'))
```

Comment:
Currently doesn't support multi-helper channel pin support - should prepare for TOML support and the implementation with that (as discussed in bot channels)